src/cli.rs

```diff
@@ -0,0 +1,560 @@
+use clap::{Parser, Subcommand};
+use iso_currency::Currency;
+
+fn parse_hex(data: &str) -> Result<Vec<u8>, hex::FromHexError> {
+    hex::decode(data)
+}
+
+fn parse_currency(data: &str) -> Result<Currency, String> {
+    Currency::from_code(data).ok_or_else(|| format!("Invalid currency '{data}'"))
+}
+
+#[derive(Debug, Parser)]
+pub(crate) struct MyOptions {
+    /// Path to the wallet directory
+    pub(crate) wallet_dir: Option<String>,
```

Comment:
I should have moved this in the last commit. Gonna figure out how to do that cleanly now.